GetVideoUrl/biliDownLoad.py

```diff
@@ -51,9 +50,18 @@ def GetSign(params,appkey,AppSecret=None):
     m = hashlib.md5()
     m.update((data+AppSecret).encode('utf-8'))
     return data+'&sign='+m.hexdigest()
+
+def ChangeFuck(params):
```

Comment:
I personally do not suggest any aggressive word to be used in open-source projects.

FYI.